exporter/coralogixexporter/config_test.go

```diff
@@ -0,0 +1,83 @@
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package coralogixexporter
+
+import (
+	// "context"
+	// "path"
+	"testing"
+	// "github.com/stretchr/testify/assert"
+	// "go.opentelemetry.io/collector/component/componenttest"
+	// "go.opentelemetry.io/collector/config"
+	// "go.opentelemetry.io/collector/config/configtest"
+	// "go.opentelemetry.io/collector/exporter/exporterhelper"
+	// "go.opentelemetry.io/collector/model/pdata"
+)
+
+func TestLoadConfig(t *testing.T) {
+	t.Logf("CORALOGIX CONFIG CHECK : Start To Test")
+
+	// factories, _ := componenttest.NopFactories()
```

Comment:
This code should either be removed or uncommented. I think this PR is a good place to add tests for configuration loading, and `TestExporter` can be added later